snorkel/labeling/lf/nlp.py

```diff
@@ -0,0 +1,202 @@
+from typing import Any, Callable, List, Mapping, NamedTuple, Optional
+
+from snorkel.labeling.preprocess import BasePreprocessor
+from snorkel.labeling.preprocess.nlp import EN_CORE_WEB_SM, SpacyPreprocessor
+
+from .core import LabelingFunction
+
+
+class SpacyPreprocessorParameters(NamedTuple):
+    """Parameters need to construct a SpacyPreprocessor."""
```

Comment:
should there be an explanation of the params here? or maybe a reference to the class where we use them?